Deployment Guides/Provisioning-Islandora-on-Ubuntu-16.md

```diff
@@ -0,0 +1,1788 @@
+```
+
+You will be prompted three times for interactive installation, once from MySQL (provide a 'root' password), once from Postfix (choose 'no configuration') and once from Oracle Java (say yes to the license). Be sure to update your `islandora-install.properties` file with the MySQL password (DB_ROOT_PASSWORD). There will be an error from installing Tomcat7 (it won't start correctly). Ignore it at this time.
+
+You should run `update-alternatives --config php` and make sure it is set to `php7.1`. Else Drupal will not install. This is because we are using the 7.1 Apache module and 7.2 material may well have been installed and set to the default for the OS during the `apt-get install`.
+
+Run `mysql_secure_installation`. You will be prompted for the root password and answers to various questions. If you have created a strong root password at this time, feel free to enable the password validation plugin and set the strength requirements appropriately. You can always do this again later if you wish to use a weak password during the setup phase. Answer yes to all the other questions.
+
+### Software Dependencies Compiled from Source <a id="from-source"></a> 
+
+ffmpeg build location
+```
+mkdir ~/ffmpeg-source
+cd ~/ffmpeg-source
+```
+#### libvpx <a id="libvpx"></a>
+```
+git clone https://chromium.googlesource.com/webm/libvpx.git
+
+cd libvpx
+
+git checkout 8366a6e4ba95e6d5af040815d2afbb4bfe628d3f
+
+./configure --disable-examples --disable-unit-tests
+
+make
+
+checkinstall --pkgname=libvpx --pkgversion="1:$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default
+```
+
+#### ffmpeg <a id="ffmpeg"></a>
+```
+cd ~/ffmpeg-source
+
+wget http://www.ffmpeg.org/releases/ffmpeg-3.3.7.tar.gz
+
+tar xf ffmpeg-3.3.7.tar.gz && rm -rf ffmpeg-3.3.7.tar.gz  
+
+cd ffmpeg-3.3.7
+```
+
+Had to do this to remove strange characters that cause build errors.
+```
+sed -i 's/×/x/' doc/filters.texi
+
+sed -i 's/×/x/' doc/ffmpeg.texi
+
+./configure --enable-gpl --enable-libass --enable-libfdk-aac --enable-libfdk-aac --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-librtmp --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-nonfree --enable-version3 --enable-libopus
+
+make
+
+checkinstall --pkgname=ffmpeg --pkgversion="7:$(date +%Y%m%d%H%M)-git" --backup=no --deldoc=yes --fstrans=no --default
+
+make install
+
+hash -r
+```
+
+The following should tell apt to not upgrade ffmpeg (we need version 1.1.1):  
```

Comment:
Can remove the "(we need version 1.1.1)" reference.

Reply:
Done.